### ISSUE TITLE
Debugging OpenGL on Intel Graphics

### ISSUE THREAD

**Issue by HoxMoxNox** [OpenKinect/libfreenect2#349]
Is there a debug option for the ./Protonect gl option? I have cl and the cpu options working and displaying feeds on all the windows, but only have RGB working on the OpenGL option, so I doubt it's a USB3 problem. I need the opengl version to work for a separate program, so I'm trying to figure out the issue it's having.

**Comment by xlz** [comment]
The OpenGL feature is known to have trouble with Intel graphics. Just use OpenCL for the moment.

**Comment by xlz** [comment]
I have posted all OpenGL fixes needed for Intel in #352.

**Comment by xlz** [comment]
Actually there is. For more debug options, use environment variables:

```
LIBGL_DEBUG=verbose MESA_DEBUG=y INTEL_DEBUG=fbo,... ./Protonect
```
`INTEL_DEBUG` has many suboptions, see http://www.mesa3d.org/envvars.html for a full list.

**Comment by floe** [comment]
Is this fixed with 0.1-rc1? If not, please re-open.
